packages/docusaurus/templates/classic/pages/index.js

```diff
@@ -6,26 +6,90 @@
  */
 
 import React from 'react';
+import cx from 'classnames';
 import Layout from '@theme/Layout';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import withBaseUrl from '@docusaurus/withBaseUrl';
-import './index.css';
+import landingPage from './data';
+import s from './s.module.css';
 
 /* Note that this is only temporary. TODO: better welcome screen */
 function Home() {
+  const context = useDocusaurusContext();
+  const {siteConfig = {}} = context;
   return (
-    <Layout title="Hello">
-      <div className="App">
-        <header className="App-header">
+    <Layout
+      /** this title will overwrite the one in config */
+      title={`Hello from ${siteConfig.title}`}
+      description="Description will go into a meta tag in <head />">
+      <header className={cx('hero hero--dark', s.header)}>
+        <div className="container">
+          <h1 className="hero__title">{siteConfig.title}</h1>
+
+          <p className="hero__subtitle">{siteConfig.tagline}</p>
+          <div className={s.buttons}>
+            <button
+              type="button"
+              className={cx(
+                'button button--outline button--primary button--lg',
+                s.getStarted,
+              )}>
+              Get Started
+            </button>
+            <iframe
+              src="https://ghbtns.com/github-btn.html?user=facebook&amp;repo=docusaurus&amp;type=star&amp;count=true&amp;size=large"
+              frameBorder={0}
+              scrolling={0}
+              width={160}
+              height={30}
+              title="GitHub Stars"
+            />
+          </div>
           <img
             src={withBaseUrl('img/logo.svg')}
-            className="App-logo"
+            className={s.logo}
             alt="logo"
           />
-          <p>
-            Edit <code>pages/index.js</code> and save to reload.
-          </p>
-        </header>
-      </div>
+        </div>
+      </header>
+      <main>
+        {landingPage.highlights && landingPage.highlights.length && (
```

Comment:
I'm probably naming them poorly 😅 Here's my intention:

Highlights are used as an example for users to put their mostly valuable features as showcase

The section that follows, I should probably rename it into 'guides' in our use case, is intended to include a simple guide + reference to help users set up their repo. The content should be updated as well. 

Named it "features" because a lot of doc sites (e.g., [Flipper](https://fbflipper.com/), [Docz](https://www.docz.site/)) often use this section to introduce features.

Reply:
On a second thought, maybe I should remove the features section for now, and add it back when we have a more complete set of docs to compose a nice story from?